packages/open-next/src/core/routingHandler.ts

```diff
@@ -185,12 +220,8 @@ export default async function routingHandler(
     !isApiRoute &&
     !isNextImageRoute &&
     // We need to check again once all rewrites have been applied
-    !staticRegexp.some((route) =>
-      route.test((internalEvent as InternalEvent).rawPath),
-    ) &&
-    !dynamicRegexp.some((route) =>
-      route.test((internalEvent as InternalEvent).rawPath),
-    )
+    !staticRouteMatcher(internalEvent.rawPath) &&
```

Comment:
comment: !&& !&& !&& is equivalent to !( || || ||) and the later might be more readable? (same for l 194)